packages/x-components/src/x-modules/popular-searches/components/popular-search.vue

```diff
@@ -5,14 +5,15 @@
     :suggestionSelectedEvents="events"
     data-test="popular-search"
     feature="popular_search"
+    #default="baseScope"
   >
-    <template #default="{ suggestion }">
-      <!--
-        @slot Popular Search's content
-            @binding {Suggestion} suggestion - Popular Search suggestion data
-      -->
-      <slot :suggestion="suggestion">{{ suggestion.query }}</slot>
-    </template>
+    <!-- eslint-disable max-len -->
+    <!--
+        @slot Popular Search content
+          @binding {Object} v-bind - `BaseSuggestion` default slot scope: **suggestion** <code>Suggestion</code> - Suggestion data<br /> **query** <code>string</code> - The query that the suggestion belongs to<br /> **filter** <code>Filter \| undefined</code> - Suggestion's filter
+    -->
+    <!-- eslint-enable max-len -->
+    <slot v-bind="{ ...baseScope }" />
```

Comment:
necessary to create a copy?

Reply:
Sadly yes, if I don't create a copy the slot docs are not generated

Reply:
WTF!

Reply:
Proof:
![image](https://user-images.githubusercontent.com/68222542/206410065-13aad1f0-f0f3-4137-84e0-df7f6972a4dc.png)
![image](https://user-images.githubusercontent.com/68222542/206410097-d14f6b9f-26e5-4046-bb32-9164becd119e.png)